project_name/settings.py

```diff
@@ -12,6 +12,8 @@
 
 import os
 
+import dj_database_url
+
```

Comment:
Separating the standard Django library and third-party packages.

Do you think we need a comment over there?